pkg/client/tx/client.go

```diff
@@ -375,6 +375,10 @@ func (tClient *txClient) addPendingTransactions(
 // Parameters:
 // - ctx: Context for managing the function's lifecycle and child operations.
 func (tClient *txClient) goSubscribeToOwnTxs(ctx context.Context) {
+	// Create a cancellable child context for managing the EventsSequence lifecycle.
+	// Cancelling it at the end of the function, when listening for submitted
+	// transactions is no longer needed will ensure that the EventsSequence, its
```

Comment:
`is no longer needed will`

Is this missing an `and`?

Reply:
It should not. What about:
```
// Cancelling it when listening for submitted transactions is no longer needed
// will ensure that the EventsSequence, its subscriptions and underlying channels
// are effectively closed.
```

Removed the `Cancelling it at the end of the function` since the `defer cancel()` is not far from the comment.

Reply:
LGTM